pkgs/collection/lib/src/unmodifiable_wrappers.dart

```diff
@@ -193,12 +193,4 @@ abstract mixin class UnmodifiableMapMixin<K, V> implements Map<K, V> {
   /// operations that change the map are disallowed.
   @override
   void clear() => _throw();
-
-  /// Throws an [UnsupportedError];
-  /// operations that change the map are disallowed.
-  set first(_) => _throw();
```

Comment:
Technically this is breaking to remove these?

We might want to just deprecate them 🤷‍♂️ . But should at least verify we don't know of any usages (I would expect none though since nobody really uses this type directly in APIs).

Reply:
I'm happy to deprecate first and check for any real usage asynchronously.